utils/find-unused-ll-keys.js

```diff
@@ -0,0 +1,97 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require("fs")
```

Comment:
This is cool!

Reply:
It has flagged a lot of unused keys and it'd be good to have them cleaned up eventually. Now that we have this on main, I think we'll be more vigilant.